tests/mock_smtp.py

```diff
@@ -33,6 +34,14 @@
 class MockSmtpServer(smtpd.SMTPServer):
     mailQueue = queue.Queue()
 
+    def __init__(self, localaddr, remoteaddr, decode_data=False):
+        kwargs = {}
+        if sys.version_info[0:2] >= (3, 5):
```

Comment:
Mind :boom:.

Actually, that makes perfect sense. I just copied the code that `six` uses; someone should tell them too.